run-tests.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
+echo '==> [INFO] Checking for broken links...'
+awesome_bot --allow-dupe --skip-save-results --allow-redirect --white-list https://reana.cern.ch **/*.html
```

Comment:
Will this fail in case of broken links? Don't we need the following?
```sh
# Quit on errors
set -o errexit

# Quit on unbound symbols
set -o nounset
```